vite.config.js

```diff
@@ -8,7 +8,12 @@ config.LCH = config.COLOR_FN !== '"oklch"'
 export default defineConfig({
   define: config,
   build: {
-    assetsDir: '.'
+    assetsDir: '.',
+    rollupOptions: {
+      output: {
+        chunkFileNames: 'model.js'
```

Comment:
Just `model.js` will not have cache busters. We need to have something like `dits/model-h5h6f.js` as we have for main JS file.

Reply:
Added the hash back in